Week3Project/Week3Project/users.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Week3Project
+{
+    class User
+    {
+        public string firstName { get; set; }
+        public string lastName { get; set; }
+        public int favoriteNumber { get; set; }
+        public int numberOfPets { get; set; }
+        public string namesOfPets { get; set; }
```

Comment:
Could this be re-worked to be of type `List<string>` instead of a single `string` to store multiple pet names?